params/config.libevm.go

```diff
@@ -231,3 +205,48 @@ func (r *Rules) extraPayload() *pseudo.Type {
 	}
 	return r.extra
 }
+
+func (c *ChainConfig) extraDescription() string {
+	if registeredExtras == nil || c.extra == nil {
+		return ""
+	}
+
+	type Descriptioner interface {
+		Description() string
+	}
+	d, ok := c.extra.Interface().(Descriptioner)
+	if !ok {
+		return ""
+	}
+	return d.Description()
+}
+
+func (c *ChainConfig) extraCheckCompatible(newcfg *ChainConfig, headNumber *big.Int, headTimestamp uint64) *ConfigCompatError {
+	if registeredExtras == nil || c.extra == nil {
+		return nil
+	}
+
+	type CompatibleChecker interface {
+		CheckCompatible(*ChainConfig, *big.Int, uint64) *ConfigCompatError
```

Comment:
Does it make sense for it to receive the old `ChainConfig` too? I assumed it would, but I guess it's YAGNI.

Reply:
Agree with yagni for now.